tutorials/tts/FastPitch_ChineseTTS_Training.ipynb

```diff
@@ -948,40 +932,50 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!(python3 NeMoChineseTTS/hifigan_finetune.py --config-path . --config-name hifigan.yaml \\\n",
-    "    model.max_steps=10 \\\n",
+    "!CUDA_VISIBLE_DEVICES=0 python3 NeMoChineseTTS/hifigan_finetune.py --config-path . --config-name hifigan.yaml \\\n",
```

Comment:
Why enforce only one GPU? Later on in the command `trainer.devices=-1`.

Reply:
it is for demonstration purposes. For example, my dev node has 3 GPUs but only 0,1 have a larger capacity of memory. Without it, model finetuning would crash.

Reply:
I can remove `trainer.devices=-1` for this tutorial.